app/controllers/admin/application_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Admin::ApplicationController < ApplicationController
+  include AuthHelper
```

Comment:
why it is w/o the special directory, like in the example `include Concerns::AuthHelper`

Reply:
Because that's how zeitwerk works :)